apps/crn-server/src/routes/research-outputs.route.ts

```diff
@@ -27,11 +31,31 @@ export const researchOutputRouteFactory = (
   researchOutputRoutes.get(
     '/research-outputs',
     async (req, res: Response<ListResearchOutputResponse>) => {
-      const { query } = req;
+      const { query, loggedInUser } = req;
+
+      const { associationId, status, ...options } = query.status
+        ? validateResearchOutputDraftFetchOptions(query)
+        : { ...validateFetchOptions(query), associationId: '', status: false };
+
+      const hasTeamRole =
+        getUserRole(loggedInUser as UserResponse, 'teams', [associationId]) !==
+        'None';
+      const hasWorkingGroupRole =
+        getUserRole(loggedInUser as UserResponse, 'workingGroups', [
+          associationId,
+        ]) !== 'None';
 
-      const options = validateFetchOptions(query);
+      if (status && !hasTeamRole && !hasWorkingGroupRole) {
```

Comment:
what is status?

Reply:
status is a query param, as we only want to do association check for drafts, i make it clear with the following 
```suggestion
      if (status === 'draft' && !hasTeamRole && !hasWorkingGroupRole) {
```